### PR TITLE
Typos from Perl5 (2011- Episode 4)

### PR DESCRIPTION
**Pull request by Fourchaux**
Some typos from https://perl5.git.perl.org/perl.git/search/HEAD?s=typo;st=free

### CODE REVIEW

**Comment by Fourchaux** [comment]
Question:

**were'nt->wasn't**

should be
**were'nt->weren't**, right?

**Comment by larsoner** [comment]
Yes I think so

**Comment by larsoner** [comment]
merge conflict

**Comment by Fourchaux** [comment]
Any corrections to make?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10101,6 +10144,7 @@ occuring->occurring
 occurr->occur
 occurrance->occurrence
 occurrances->occurrences
+occurrent->occurred
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `occurrent` is a valid word  
https://www.vocabulary.com/dictionary/occurrent

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10790,6 +10835,7 @@ peoms->poems
 peopel->people
 peotry->poetry
 pepare->prepare
+peported->reported
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Also `purported`

**Pull request merged**